cybozu/tcp.cpp

```diff
@@ -445,7 +445,7 @@ setup_server_socket(const char* bind_addr, std::uint16_t port, bool freebind) {
     }
     freeaddrinfo(res);
 
-    if( listen(s, 128) == -1 ) {
```

Comment:
I could not find a reference that `listen` can accept -1 as a valid backlog.
Could you give me a pointer?

Reply:
Sorry, I might be misinterpreting https://pubs.opengroup.org/onlinepubs/9699919799/functions/listen.html
Perhaps settings this to a reasonably high value would be more reliable..

Reply:
Sounds good.  What do you think about 4096?

Reply:
Let's go with that. The default for Linux 5.4 is 4096 so it should be reasonable for a while.